tests/clients/test_job.py

```diff
@@ -84,7 +99,47 @@
 
 JOB_SUMMARY = JobSummary(id=JOB_ID, metadata=JOB_METADATA)
 JOB_SUMMARY_BODY = {"jobId": str(JOB_ID), "meta": JOB_METADATA_BODY}
-
+INSTANCE_SIZE = InstanceSize(milli_cpus=MILLI_CPUS, memory_mb=MEMORY_MB)
+INSTANCE_SIZE_BODY = {"milliCpus": str(MILLI_CPUS), "memoryMb": str(MEMORY_MB)}
+JOB_PARAMETERS = JobParameter(
+    name="job name",
+    param_type=ParameterType.TEXT,
+    default="default parameter",
+    required=True,
+)
+JOB_PARAMETERS_BODY = {
+    "name": JOB_PARAMETERS.name,
+    "type": "text",
+    "default": JOB_PARAMETERS.default,
+    "required": str(True),
+}
+JOB_COMMAND = JobCommand(name="command name", parameters=[JOB_PARAMETERS])
+JOB_COMMAND_BODY = {
+    "name": JOB_COMMAND.name,
+    "parameters": [JOB_PARAMETERS_BODY],
+}
+JOB_DEFINITION = JobDefinition(
+    working_dir="working dir",
+    command=JOB_COMMAND,
+    image_type="image type",
+    conda_environment="conda environment",
+    environment_ids=[str(ENVIRONMENT_ID)],
+    instance_size_type="instance size type",
```

Comment:
As above:
```suggestion
    instance_size_type="m4.xlarge",
```
It's also the case that `instanceSizeType` must be `"custom"` if `instanceSize` is set. Should we have two tests - one with a 'custom' instance size and one with a preset like 'm4.xlarge' suggested above?